components/overflow-group/overflow-group-mixin.js

```diff
@@ -130,8 +131,13 @@ export const OverflowGroupMixin = superclass => class extends LocalizeCoreElemen
 			}
 		});
 
+		const containerStyles = {
```

Comment:
This sets the min and max height of the container to be the tallest slotted item as calculated in _getItemLayouts. I should also be checking overflowContainer height as it's possible that could be taller.

This is because the overflow groups always appear on one line and this prevents a momentary flash where the contents wrap to two line, which can be jarring.